code/modules/mob/living/living.dm

```diff
@@ -1280,16 +1280,14 @@
 	..()
 	update_z(new_z)
 
-/mob/living/MouseDrop(mob/over)
+/mob/living/MouseDrop_T(atom/dropping, atom/user)
+	var/mob/living/U = user
```

Comment:
you may need to check if user is null when cast to /mob/living

Reply:
also may need to check for proximity

Reply:
im not too familiar with mousedrop_t, how could user end up null there? i tried clickdragging mobs onto items/objects/etc and vise versa after reading this to be sure, no runtimes

Reply:
yes i pulled your branch and i agree it seems to be good, its not a problem from what i see its all good there

Reply:
then mob/living/MouseDrop_T(atom/dropping, mob/living//user) ?